ttnn/ttnn/operations/core.py

```diff
@@ -253,7 +259,8 @@ def to_torch(
     cq_id: Optional[int] = 0,
 ) -> "torch.Tensor":
     """
-    Converts the `ttnn.Tensor` tensor into a `torch.Tensor`.
+    Converts the `ttnn.Tensor` tensor into a `torch.Tensor`. It does not call to_layout for bfloat8_b or bfloat4_b as we now convert
+    to tile layout during when calling to_torch.
```

Comment:
Isn't this doc string for to_torch? Isn't the below code not calling `tensor.to`?

Maybe change to something like
```
... It does not convert to row major layout for bfloat8_b or bfloat4_b.
```
or
```
... It does not convert to row major layout for bfloat8_b or bfloat4_b because those need tile layout.
```

Reply:
I meant 
"It does not call to_layout for bfloat8_b or bfloat4_b as we now convert
    to tile layout during when calling tensor.to_torch"
I'll make it clearer,